### PR TITLE
add section on project maintainers

### PR DESCRIPTION
**Pull request by smola**
Drafted taking into account notes from meetings on project maintainership. I filled some gaps that were not discussed in depth during the meeting, so feel free to revoke them if there is no agreement on them.

### CODE REVIEW

File: engineering/maintainers.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Project Maintainers
+
+## General Guidelines
+
+* Every GitHub repository must have an assigned maintainer.
+* Maintainer is specified in a MAINTAINER file in the top
```
Comment (Serabe) [inline engineering/maintainers.md]: s/MAINTAINER/MAINTAINERS/

File: engineering/maintainers.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  * Code Review pull requests and ensure that issues get
+    handled.
+  * Ensure that good practices are followed and that there
+    is consistency among the code base.
+
+## Code Review
+
+* A maintainer should review every PR but can ask for
+  additional reviewers as needed (e.g. someone else is
+  more knowledgeable about as change or has a say as
```
Comment (Serabe) [inline engineering/maintainers.md]: s/about as/about a/

File: engineering/maintainers.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  stakeholder) or completely delegate the review.
+* Non-maintainers at source{d} are discouraged to do code
+  review on PRs they are not asked to as a general practice.
+  However, they are allowed to at their discrection if they
```
Comment (Serabe) [inline engineering/maintainers.md]: s/discrection/discretion/

File: engineering/maintainers.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  think it's appropriate on a specific case.
+* Deadline for each code review is 1 working day.
```
Comment (Serabe) [inline engineering/maintainers.md]: Desirable deadline.

File: engineering/maintainers.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+* If there is a disagreement among the PR author and multiple
+  reviewers and consensus cannot be reached or [it's not worth to reach](http://bikeshed.org/)
+  the maintainer has the final word.
+* A maintainer can merge his own PRs without code review,
```
Comment (Serabe) [inline engineering/maintainers.md]: Not sure we have talked exactly about this. They can only do so in PRs regarding maintainance (typos, tags, Makefile...).

File: engineering/maintainers.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+  * Ensure that good practices are followed and that there
+    is consistency among the code base.
+
+## Code Review
+
+* A maintainer should review every PR but can ask for
+  additional reviewers as needed (e.g. someone else is
+  more knowledgeable about as change or has a say as
+  stakeholder) or completely delegate the review.
+* Non-maintainers at source{d} are discouraged to do code
```
Comment (abeaumont) [inline engineering/maintainers.md]: I'd change _discouraged_ by _not expected_, but that's just my opinion

File: engineering/maintainers.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
+# Project Maintainers
+
+## General Guidelines
+
+* Every GitHub repository must have an assigned maintainer.
+* Maintainer is specified in a MAINTAINER file in the top level directory as follows: `Full Name <email> (@github_handle)`
```
Comment (vmarkovtsev) [inline engineering/maintainers.md]: MAINTAINERS?

**Comment by Serabe** [comment]
@smola a change in formatting hide my comments but they still apply, can you review them please? Thanks!

**Comment by smola** [comment]
@Serabe Done.

**Pull request merged**